### ISSUE TITLE
Extensible enums are not infered by static type checking

### ISSUE THREAD

**Issue by vukisz** [milnet92/MXTXppInterpreter#16]
Is it only me, or did I do some kind of typo in here?

```
PriceDiscProductCodeType productCodeType;

productCodeType = PriceDiscProductCodeType::Table;
```
Shows parsing error:
![image](https://github.com/user-attachments/assets/b0298011-d7f3-4348-a67b-d94d3cc46810)

FYI PriceDiscProductCodeType is extensible enum:
![image](https://github.com/user-attachments/assets/3066ca87-212f-4016-9b33-07e012f54902)
![image](https://github.com/user-attachments/assets/2e402883-e9fc-4772-b7ce-7663f1f3dced)

**Comment by milnet92** [comment]
Issue was resolved with b60b107. Please check.

**Comment by vukisz** [comment]
That was fast! Thank you it works after loading and building latest MXppTools
